tests/io/eager/test_read.py

```diff
@@ -145,3 +145,16 @@ def test_read_table_simple_list_for_cols_cats(dataset, store_session):
     expected_df = expected_df.astype("category")
 
     pdt.assert_frame_equal(df, expected_df, check_dtype=False, check_like=True)
+
+
+def test_write_and_read_default_table_name(store_session):
```

Comment:
you will want to add/change tests that check that this behavior also works for the `read_table` functions. You may want to change https://github.com/JDASoftwareGroup/kartothek/blob/dcee179b7a69d022f5001bd309deba806c235c31/tests/io/dask/delayed/test_read.py#L27 so that no `table` kwarg is passed (and probably replace the test code which defines the tables as "`core`" to `SINGLE_TABLE`).

Reply:
I added a new commit where I removed the table kwarg in above code snippet and refactored tests as far as required to use `SINGLE_TABLE` instead of `core`.